templates/liveblog-loop-amp.php

```diff
@@ -0,0 +1,21 @@
+<?php if ( $show_archived_message ): ?>
+<div class="liveblog-archived-message">
+	<?php printf( __( '<strong>This liveblog is archived.</strong> If you need to publish new updates or edit or delete the old ones, you need to <a href="%s">enable it first</a>.' , 'liveblog'), get_edit_post_link() . '#liveblog' ); ?>
```

Comment:
We should consider `get_edit_post_link()` to be untrusted and escape it's output.